corehq/ex-submodules/couchexport/shortcuts.py

```diff
@@ -69,10 +69,14 @@ def export_response(file, format, filename, checkpoint=None):
             logging.exception("Error with filename: %r" % filename)
             filename = "data"
         finally:
-            response['Content-Disposition'] = 'attachment; filename="{filename}.{format.extension}"'.format(
+            header_value = 'attachment; filename="{filename}.{format.extension}"'.format(
                 filename=filename,
                 format=format
             )
+            # HTTP headers shouldn't contain newlines
+            header_value.replace("\n", " ")
+            header_value.replace("\r", " ")
+            response['Content-Disposition'] = header_value
```

Comment:
I wonder if we can have a more centralized/consistent way to do this? If you search the codebase for `attachment; filename=` you can find all sorts of different levels of cleanup done to workaround various bugs of this flavor.

I think the most comprehensive version is probably in the main exports code [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/views.py#L574). wonder if we can extract that to a utility and use it here (and ideally, more widely).